seungjong/app.js

```diff
@@ -1,174 +1,33 @@
-require('dotenv').config();
-const cors = require("cors");
-const logger = require("morgan");
+// // Layered Patteren
+const http = require('http');
 const express = require('express');
-const { DataSource } = require("typeorm");
+const cors = require('cors');
+const morgan = require('morgan');
+
+const dotenv =require("dotenv")
+dotenv.config();
```

Comment:
```suggestion
require("dotenv").config();
```
환경 변수를 불러오는 코드는 위와같이 한줄로 쓰일 수 있으며, 해당 코드는 가능하면 server 실행시 읽어부르는 파일의 최상단에 작성될 수 있도록 해주세요. 맨 윗줄로 옮겨주세요.